.github/bot_templates/MARKER_REPORT_ISSUE.md.j2

```diff
@@ -0,0 +1,34 @@
+---
+title: AWS Marker report
+---
+# AWS marker report
+
+- Repository: {{ data.meta.repo_url }}
+- Reference Commit: `{{ data.meta.commit_sha }}`
+- Timestamp: `{{ data.meta.timestamp }}`
+
+This is an autogenerated report on our pytest marker usage with a special focus on our AWS compatibility markers, i.e. the ones prefixed with `aws_`.
+
+```text
+{% for name, count in data.aggregated.items() -%}
+{{ name }} : {{ count }}
+{% endfor -%}
+``` 
+
+Both aws_unknown and `aws_needs_fixing` need to be reduced to `0` over time.
```

Comment:
```suggestion
Both `aws_unknown` and `aws_needs_fixing` should be reduced to `0` over time.
```
I think it might be better to phrase this a bit more lightly.